tests/test_unified_storage_cleanup.py

```
@@ -0,0 +1,152 @@
+import pytest
+
+from fetchtastic.constants import APP_DIR_NAME
+from fetchtastic.download.cache import CacheManager
+from fetchtastic.download.client_app import MeshtasticClientAppDownloader
+from fetchtastic.download.interfaces import Asset, Release
+from fetchtastic.download.version import VersionManager
+
+pytestmark = [pytest.mark.unit, pytest.mark.core_downloads]
+
+
+def _make_downloader(tmp_path):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_make_downloader`

(ANN202)